client/ayon_houdini/plugins/publish/collect_usd_render.py

```diff
@@ -68,3 +69,12 @@ def replace_to_f(match):
 
         # stub required data for Submit Publish Job publish plug-in
         instance.data["attachTo"] = []
+
+    def evalParmNoFrame(self, rop, parm, **kwargs):
+        try:
+            return evalParmNoFrame(rop, parm, **kwargs)
+        except Exception as exc:
+            raise PublishError(
+                f"Failed evaluating parameter '{parm}' on Rop node: {rop.path()}",
+                detail=f"{exc}"
+            )
```

Comment:
What do you think of implementing this on `HoudiniInstancePlugin` base class?

Reply:
That's fair

Reply:
https://github.com/ynput/ayon-houdini/pull/190/commits/0363a57b699b214a901ac79deab352d856f93e7e